runtime/polkadot/src/weights/pallet_utility.rs

```diff
@@ -42,20 +43,20 @@ use sp_std::marker::PhantomData;
 /// Weight functions for `pallet_utility`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
-	fn batch(c: u32) -> Weight {
-		(13_489_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((605_000 as Weight).saturating_mul(c as Weight))
+	fn batch(c: u32, ) -> Weight {
+		(16_110_000 as Weight)
+			// Standard Error: 2_000
+			.saturating_add((6_325_000 as Weight).saturating_mul(c as Weight))
```

Comment:
10x increase

Reply:
This file wasn't updated since 1 year.

Here we have similar results: https://github.com/paritytech/polkadot/pull/4113/files#diff-877433e4f55fc21b56c123e9858f101eda2bed860ddec10d6bdbecf9510ff812R50